app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,7 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".app.users.UserEditView">
+        <activity android:name=".app.users.UserEditActivity">
```

Comment:
Now that I think about it, this is a much better name.  Because the UserEditActivity isn't the view, and that's kind of the whole point.  an Activity is a collection of 'stacks' of M/V/VM components